apps/judicial-system/web/src/routes/DetentionRequests/DetentionRequests.tsx

```diff
@@ -42,7 +42,7 @@ export const CasesQuery = gql`
 `
 
 export const DetentionRequests: React.FC = () => {
-  const [cases, setCases] = useState<Case[]>(null)
+  const [cases, setCases] = useState<Case[] | null>(null)
```

Comment:
Should we not use
`useState<Case[]?>(undefined)`?

Reply:
That's not a valid syntax 👇

![Screenshot 2020-11-10 at 13 37 22](https://user-images.githubusercontent.com/3789875/98680910-f0909200-2359-11eb-82e4-47866c3e39a2.png)